### PR TITLE
Add checks for proper config values in deployment validation

### PR DESCRIPTION
**Pull request by maurelian**
This PR contains the most complete version of the deployment validation scripts which were run during the 0.5.0 regenesis.

Since the regenesis I've made quite a few small adjustments to improve the UX, but all the checks present here were also done prior to finalizing the upgrades.

### CODE REVIEW

File: packages/contracts/tasks/validate-chugsplash-dictator.ts [hunk 1]
```diff
@@ -128,9 +131,11 @@ Now validating the Chugsplash Dictator deployment at\n${getEtherscanUrl(
       }
     )
     await getInput(c.yellow('OK? Hit enter to continue.'))
+    console.log()
 
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const bridgeArtifact = require('../artifacts/contracts/L1/messaging/L1StandardBridge.sol/L1StandardBridge.json')
+    // // eslint-disable-next-line @typescript-eslint/no-var-requires
+    // const bridgeArtifact = require('../artifacts/contracts/L1/messaging/L1StandardBridge.sol/L1StandardBridge.json')
```
Comment (tynes) [inline packages/contracts/tasks/validate-chugsplash-dictator.ts:137]: Did you mean to delete this line?

Comment (maurelian) [reply packages/contracts/tasks/validate-chugsplash-dictator.ts]: I did yes!

**Comment by tynes** [comment]
Still looking for review on this?

**Comment by maurelian** [comment]
@tynes yes please!

**Comment by maurelian** [comment]
Also nudge to @smartcontracts ^

**Review by tynes — APPROVED** [review]

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**